README.md

```diff
@@ -65,6 +65,21 @@ Tests for this contract are implemented in Haskell using the [cleveland framewor
 For more detail about the test suite and how to execute it, check out its
 [README](./haskell/test/README.md).
 
+## Contract Metadata
```

Comment:
It's a bit odd that this is here BTW, I think it should be put perhaps before the `Deploying/Originating` section (here it's too late IMO).

Also, this should be linked to in the `building` doc, I think, given that the metadata needs to be provided when building the storage.

In fact, maybe it makes sense to move this section to that document entirely? What do you think?

Reply:
I decide to move this section completely to the `building.md`, also applies your suggestion over there. Let me know if this is alright.